### DISCUSSION TITLE
OpenID Connect not working in 10.3.2

### DISCUSSION THREAD

**Discussion by khenow1978** [pimcore/pimcore#11622] [General]
Dear all,

we are currently using the Enterprise edition of Pimcore X and just recently upgraded from 10.2.5 to 10.3.1 (and today to 10.3.2). Suddenly the "open id connect" Button doesn't work anymore. 

I see the button on the login page but it's not having any function. In DEV we are still using 10.2.5 and the button works perfectly fine. That's why I assume that this weird behaviour has something to do with the upgrade.

I'm not sure how to proceed here because there's no real error message anywhere in the system. Any hint would be appreciated.

**Comment by khenow1978** [comment]
Solved!
In this screenshot there's an "a" indicated:

![image](https://user-images.githubusercontent.com/92732594/157248712-0f674f82-0968-40ec-a214-0e92e26d3162.png)

in 10.3.2 there's "button" indicated at that location. However if I change it back to "a" it works fine.

**Reply by fashxp** [reply]
Hmm strage, button works fine for me.
What browser are you using?

**Reply by khenow1978** [reply]
I tried Chrome & Firefox but both showed the same behaviour. Something odd: When I hover over the button the cursor changed to this little hand with the index finger (showing it's clickable) but there was no hyperlink indicated.. as if the target was missing. On DEV I hovered over the button and firefox provided the "copy link to clipboard" menu and showed it at the buttom of the window.

**Reply by fashxp** [reply]
please update openid-connect package to v1.0.4 ... should be fixed now.

**Reply by khenow1978** [reply]
done. Works fine. :)
